### PR TITLE
Update for Zip64 compatibility

### PR DESCRIPTION
**Pull request by subsetpark**

### CODE REVIEW

**Comment by ananthakumaran** [comment]
could you separate the formatting changes and the changes related to zip64. If you open a separate PR for formatting changes, I could merge it right away. is the zip64 related code being tested now? I am ok with adding something like `@force_64 !!System.get_env("ZSTREAM_FORCE_ZIP_64")` and then running the test once again in CI with the env set.

**Comment by subsetpark** [comment]
@ananthakumaran done!

File: lib/zstream/protocol.ex [hunk 1]
```diff
@@ -1,132 +1,405 @@
...
+
+  def local_file_header(%{
+        c_size: compressed_size,
+        size: uncompressed_size,
+        crc: crc32,
+        options: options,
+        name: filename
+      })
+      when compressed_size > @two_byte_max_uint or uncompressed_size > @four_byte_max_uint or
+             @force_64 do
```
Comment (ananthakumaran) [inline lib/zstream/protocol.ex]: could you explain how this is supposed to work? From my understanding, `c_size` and `size` will be always be zero during initialization as we are reading from a stream and we won't know either of these values.

**Comment by ananthakumaran** [comment]
FYI zip64 support has been added in the latest release

**Pull request closed without merging**